src/components/Credits/Credits.test.tsx

```diff
@@ -13,7 +13,9 @@ describe('component Credits', () => {
     );
     const citylabImage = screen.getByAltText(/Logo Citylab/i);
     expect(citylabImage).toBeInTheDocument();
-    expect(citylabImage.getAttribute('src')).toEqual('https://logos.citylab-berlin.org/logo-citylab-berlin-outline.svg');
+    expect(citylabImage.getAttribute('src')).toEqual(
```

Comment:
This makes the test dependent on an external resource. Are we sure this should be like this?